components/CalculAmortissements.tsx

```diff
@@ -0,0 +1,71 @@
+"use client";
+
+import { useState } from 'react';
+import { Button } from "./ui/Form/Button/Button";
+import { InputWithLabel } from "@/components/ui/Form/InputWithLabel/InputWithLabel";  
+
+const CalculAmortissements = () => {
+  const [valeurAcquisition, setValeurAcquisition] = useState<number>(0);
+  const [valeurResiduelle, setValeurResiduelle] = useState<number>(0);
+  const [dureeAmortissement, setDureeAmortissement] = useState<number>(0);
+  const [amortissementAnnuel, setAmortissementAnnuel] = useState<number>(0);
+
+  const handleInputChange = (setter: React.Dispatch<React.SetStateAction<number>>) => (e: React.ChangeEvent<HTMLInputElement>) => {
+    setter(Number(e.target.value));
+  };
+
+  const calculerAmortissement = () => {
+    const amortissement = (valeurAcquisition - valeurResiduelle) / dureeAmortissement;
+    setAmortissementAnnuel(amortissement);
+  };
+
```

Comment:
const { valeurAcquisition, valeurResiduelle, dureeAmortissement, amortissementAnnuel } = amortissements;
to use values of the state